Blocked by Windows failures 
```
Creating Layout for Visual Studio 2017 build tools: 
Success
Installing Certificate Sign Root Certificate: 
Failed
Installing Certificate Root Certificate: 
Failed
Installing Visual Studio [201](https://github.com/saltstack/salt/actions/runs/9371525183/job/25807926056?pr=66617#step:10:202)7 build tools: 
Failed
Failed to install Visual Studio Build Tools
[20:22:06] Command '('powershell.exe', '&', 'pkg/windows/build.cmd', '-Architecture', 'amd64', '-Version', '3006.8+112.g7178fc0f8f', '-PythonVersion', '3.10.14', '-RelenvVersion', '0.16.0',
           '-CICD', '-SkipInstall')' returned non-zero exit status 1.                                                                                                                        
Error: Process completed with exit code 1.
##[debug]Finishing: Build Windows Packages
```

--- a/pkg/windows/install_vs_buildtools.ps1
+++ b/pkg/windows/install_vs_buildtools.ps1
@@ -122,7 +122,7 @@ if ( $install_build_tools ) {
         Start-Process -FilePath "certutil" `
                       -ArgumentList "-addstore", `
                                     "Root", `
-                                    "$($env:TEMP)\build_tools\certificates\manifestCounterSignRootCertificate.cer" `
+                                    "$env:TEMP\build_tools\certificates\manifestCounterSignRootCertificate.cer" `
                       -Wait -WindowStyle Hidden
         if ( Test-Path -Path Cert:\LocalMachine\Root\3b1efd3a66ea28b16697394703a72ca340a05bd5 ) {
             Write-Result "Success" -ForegroundColor Green
@@ -138,7 +138,7 @@ if ( $install_build_tools ) {
         Start-Process -FilePath "certutil" `
                   -ArgumentList "-addstore", `
                                 "Root", `
-                                "$($env:TEMP)\build_tools\certificates\manifestRootCertificate.cer" `
+                                "$env:TEMP\build_tools\certificates\manifestRootCertificate.cer" `
                   -Wait -WindowStyle Hidden
         if ( Test-Path -Path Cert:\LocalMachine\Root\8f43288ad272f3103b6fb1428485ea3014c0bcfe ) {
             Write-Result "Success" -ForegroundColor Green

